luci/sqm-cbi.lua

```diff
@@ -85,7 +85,8 @@ ul = s:taboption("tab_basic", Value, "upload", translate("Upload speed (kbit/s)
 ul.datatype = "and(uinteger,min(0))"
 ul.rmempty = false
 
-dbl = s:taboption("tab_basic", Flag, "debug_logging", translate("Create log file for this SQM instance under /var/run/sqm/${Interface_name}.debug.log. Make sure to delete log files manually."))
+dbl = s:taboption("tab_basic", Flag, "debug_logging", translate("Create log file for this SQM instance under /var/run/sqm/${Interface_name}.[start|stop]-sqm.log."))
+dbl.default = "1"
```

Comment:
Don't enable debugging by default...

Reply:
First off thanks for your time.
Mmmh, I see your point. The goal here is to make logging to file low-cost enough to always enable it. I do not really see this as a debugging aid that needs to be activated, but would love a always-on log, so if people have issues, all I need is to ask them to collect the log files and post them. Do you see a route to achieving that goal or is it still too expensive?

Reply:
The problem is that we can't guarantee that the log file isn't being written to flash (which can wear out), so we probably don't want to have needless logging on by default...

If you are telling people to post the log files, can't you just tell them to turn on the logging first? ;)

Reply:
Mmmh, on OpenWrt we should end up on tmpfs/RAM as far as I can tell, so I hoped that having a toggle to disable this would be sufficient. But then, sure I can ask people to check that box if problems occur and get the logs that way. I guess I will toggle back to default off.

Reply:
Fixed...